src/SurveyDesign.jl

```diff
@@ -14,7 +13,9 @@ abstract type AbstractSurveyDesign end
 """
     SimpleRandomSample <: AbstractSurveyDesign
 
-Survey design sampled by simple random sampling.
+
+A simple random sample dataset can be loaded from a data frame into a `SimpleRandomSample` object for downstream analyses. 
```

Comment:
This sentence is too long to be the first descriptive sentence of the type. Usually these sentences are short and concise. Extra details are given below, if necessary. I think the previous sentence was more appropriate.

This is the case for stratified as well.